bookkeeper-server/src/test/java/org/apache/bookkeeper/server/http/TestHttpService.java

```diff
@@ -832,6 +833,7 @@ public void testGCDetailsService() throws Exception {
         HttpServiceRequest request1 = new HttpServiceRequest(null, HttpServer.Method.GET, null);
         HttpServiceResponse response1 = gcDetailsService.handle(request1);
         assertEquals(HttpServer.StatusCode.OK.getValue(), response1.getStatusCode());
+        assertTrue(JsonUtil.fromJson(response1.getBody(), List.class).get(0).toString().contains("ledgerDir"));
```

Comment:
Please add a detailed unit test for DbLedgerStorage, InterleavedLedgerStorage, and SortedLedgerStorage.

Reply:
I cannot simply do it here, Now, Bookkeeper use Junit test framework. it doesn't have dataProvider. I want upgrade the Junit to TestNG in another pr. I will solve this in that pr. @hangc0276